storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -112,6 +112,13 @@ include::modules/persistent-storage-csi-vsphere-disable-storage-consequences.ado
 
 include::modules/persistent-storage-csi-vsphere-disable-storage-procedure.adoc[leveloffset=+2]
 
+include::modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-overview.adoc[leveloffset=+1]
+
+:FeatureName: Increasing volumes per node
+include::snippets/technology-preview.adoc[leveloffset=+2]
```

Comment:
I'm hesitant to say that you should create a "Limitations" module for 1-2 limitations, but I can't help but wonder if the snippet would look better under the overview, and not under "Limitations." I guess TP could technically be a limitation, though (even though having it listed under "Limitations" might not be your intention).